endpoints.md

```diff
@@ -12,4 +12,42 @@
 * /api/core/sites
 
 ## Proposed Endpoints
-* /api/discover
+* /api/discover/(dso)?(sort)
+  * "context" contains the following
+    * Item counts
+    * All items accessible to user by context default sort
+    * All items accessible to user by selected sort
+    * All facets counts for context
+    * All filters available for the context
+  * Get context for the full site
+    * Get all items for a specific community or collection
+    * Get all items for a full text search
+    * Get all items for a faceted search
+    * Get all items for a complex search
+* /api/user/eperson
+* /api/user/epersongroup
+* /api/authorize/(dso)
+* /api/submit
```

Comment:
not sure. Item should be created with a POST to the /core/item endpoint.
We need to decide if we want an endpoint also for workspaceitem and workflowitem or if we want to support the workflow as a status of the item to update to trigger a workflow advance